pecoff.c

```diff
@@ -678,7 +678,7 @@ coff_add (struct backtrace_state *state, int descriptor,
   sects_view_valid = 1;
   opt_hdr = (const b_coff_optional_header *) sects_view.data;
   sects = (const b_coff_section_header *)
-    (sects_view.data + fhdr.size_of_optional_header);
+    (void *)((const char *)sects_view.data + fhdr.size_of_optional_header);
```

Comment:
...so, including the unchanged one on the previous line, that's what, 3 casts in a row here? I wonder if there's a way to make that more efficient?

Reply:
Casts don't carry any runtime cost, so there is efficiency issue here.